server/container/util-java8/src/test/java/org/apache/james/util/streams/ContainerNames.java

```diff
@@ -25,4 +25,5 @@
     String ELASTICSEARCH = "elasticsearch:2.2.2";
     String NGINX = "nginx:1.7.1";
     String TIKA = "logicalspark/docker-tikaserver:1.15rc2";
+    String SPAMASSASSIN = "dinkel/spamassassin:3.4.0";
```

Comment:
I discover that file : why it's called Names (they are not names) ?
And why in util.streams ?

Reply:
It's next to SwarmContainer. Maybe we need a **docker** package in there. And maybe **Images** might be a better name for this.

Would you agree with such a change @mbaechler  ?

Reply:
of course